draft-ietf-core-new-block.xml

```diff
@@ -1791,8 +1786,6 @@ o  Reference: [RFCXXXX]]]></artwork>
 
       <?rfc include='reference.RFC.8742'?>
 
-      <?rfc include='reference.RFC.8949'?>
```

Comment:
Maybe you still want to reference 8949 as you are using CBOR?

Reply:
Interesting question.  I dropped 8949 as we no longer referenced it as no longer using Indefinite-Length Arrays.  We still have 8742 for CBOR sequences which refers to 7049 and 8742 was not updated by 8949.

I will add 8949 back in.  Thanks for picking this up.

Reply:
Done and code pushed